spec/models/need_spec.rb

```diff
@@ -29,11 +29,14 @@
   it { is_expected.to be_versioned }
 
   it '#status' do
-    need1 = build :need, name: 'medicines', completed_on: nil
-    expect(need1.status).to eq 'To do'
+    need1 = build :need, name: 'medicines', status: :to_do
```

Comment:
I guess there should be tests for the other states too.

Reply:
Thanks, done.